src/utils.js

```diff
@@ -30,6 +30,12 @@ export function isValidUrl(input) {
     return pattern.test(input);
 }
 
+export function isValidEmail(input) {
+    const pattern = new RegExp(/^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/);
```

Comment:
Har vi lenke til dokumentasjon på denne regex'en? Slik jeg har forstått det finnes det en rekke ulike regex'er for e-postvalidering, og at ikke alle tar høyde for alle ulike e-postadressevarianter.

Reply:
Jeg ser ikke poeng med å gjøre en så voldsom validering her ? Bare pass på det enkleste, at det finnes noe både før og etter én `@`, og kanskje sjekke at det finnes et punktum mellom de to siste leddene i domene, for å hjelpe brukeren å unngå de groveste feil. Har bruker skrevet inn feil epost-adresse utover dette, så blir det ingen epost om natta..

Reply:
Først trim input, så test på `^[^@]+@[^@.]+\.[^@]+$` ? Oppsummert: bruker har putta inn én og bare én krøllalfa, og det er noe som ikke er whitespace før og etter, og det finnes en dot i det etter, fordi vi ikke kan akseptere f.eks. domener som "@localhost", som ikke er fullkvalifiserte med en TLD-komponent. Dette vil likevel slippe gjennom mye rart, men da tenker jeg: det gjør ikke noe ?